### ISSUE TITLE
`examples/kitchen-chef` centos test fails

### ISSUE THREAD

**Issue by Dispader** [inspec/inspec#854]
### Description

Both Ubuntu test suites pass for the `examples/kitchen-chef` example, but test appear to fail for CentOS:

```
(~/Development/University/github/inspec/examples/kitchen-chef) master {azazel}05:15pm] bundle exec kitchen test default-centos-71
-----> Starting Kitchen (v1.10.2)
-----> Cleaning up any prior instances of <default-centos-71>
...
-----> Verifying <default-centos-71>...
       Use `/Users/jake/Development/University/github/inspec/examples/kitchen-chef/test/integration/default` for testing

Target:  ssh://vagrant@127.0.0.1:2222

  ✔  System Package nginx should be installed
  ✖  01: Verify nginx service (1 failed)
     expected that `Service nginx` is enabled
  ✔  User nginx should exist

Summary: 4 successful, 1 failures, 0 skipped
>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: 1 actions failed.
>>>>>>     Verify failed on instance <default-centos-71>.  Please see .kitchen/logs/default-centos-71.log for more details
>>>>>> ----------------------
>>>>>> Please see .kitchen/logs/kitchen.log for more details
>>>>>> Also try running `kitchen diagnose --all` for configuration
```
### InSpec and Platform Version

Tell us which version of InSpec (`inspec version` or SHA of a commit) and Operating System distribution you are using:

default Gemfile in the current master branch:

```
# encoding: utf-8
source 'https://rubygems.org'

gem 'inspec', path: '../../.'

group :test do
  gem 'bundler', '~> 1.5'
  gem 'minitest', '~> 5.5'
  gem 'rake', '~> 10'
  gem 'rubocop', '~> 0.33.0'
  gem 'simplecov', '~> 0.10'
end

group :integration do
  gem 'berkshelf', '~> 4.0'
  gem 'test-kitchen', '~> 1.4'
  gem 'kitchen-vagrant'
  gem 'kitchen-inspec'
  gem 'concurrent-ruby', '~> 0.9'
end
```
### Replication Case

```
cd examples/kitchen-chef
bundle
bundle exec kitchen test default-centos-71
```
### Possible Solutions

None yet, but I'll ahve a look.
### Stacktrace

None, see above.

**Comment by Dispader** [comment]
It looks as though `systemctl` at least knows about it:

```
[vagrant@default-centos-71 ~]$ sudo systemctl | grep -i nginx
nginx.service                                                                            loaded active running   nginx - high performance web server
```

**Comment by chris-rock** [comment]
This may be related to #749

**Comment by kekaichinose** [comment]
Thank you for tracking down this issue. As with many large open source projects, we have a backlog of bugs and issues of varying user impact. Some issues are perpetually subject to lower priority than others in our backlog. We have reviewed the issue, and we have determined this a very low priority when considered in the context of the entire backlog. We have elected to close it as "Declined" due to competing priorities. We very much appreciate you bringing this to our attention.

Thank you!
__Chef InSpec Team__
